docs/api/rest_api/rest_api_authentication.md

```diff
@@ -18,6 +18,11 @@ For other security related subjects, see:
 - [Cross-origin requests](rest_responses.md#cross-origin)
 - [`access_control`]([[= symfony_doc =]]/security/access_control.html)
 
+!!! caution "SiteAccess login"
+
+    The anonymous user is used to perform authentification requests.
+    Therefore, the "Anonymous" role must have `user/login` permission on the SiteAccess that matches the REST domain or is passed through the [`X-Siteaccess` header](rest_requests.md#siteaccess).
```

Comment:
```suggestion
    The anonymous user is used to perform authentification requests.
    Therefore, the "Anonymous" role must have `user/login` permission on the SiteAccess that matches the REST domain or is passed through the [`X-Siteaccess` header](rest_requests.md#siteaccess).
    For example, to not have this permission on the related SiteAccess breaks Ibexa Connect app.
```

Reply:
@dabrt @lionelakpagni Or even go farther with cross-examples:
```suggestion
    The anonymous user is used to perform authentification requests.
    Therefore, the "Anonymous" role must have `user/login` permission on the SiteAccess that matches the REST domain or is passed through the [`X-Siteaccess` header](rest_requests.md#siteaccess).
    For example, to not have this permission on the related SiteAccess breaks Ibexa Connect app.
    Regarding [those examples](https://doc.ibexa.co/projects/connect/en/latest/general/connecting_to_ibexa_connect/), "Anonymous" role must have user/login permission on the SiteAccess matched by `my-site.co`, or the `campaign` SiteAccess.
```

Reply:
I feel like this is already enough (it will be mentioned in the Connect doc as well) - but if you feel like expanding this then feel free 😉